### ISSUE TITLE
Python 2.7.16 DEPRECATION error when installing extensions for CIF2Cell-1.2.10

### ISSUE THREAD

**Issue by Ezimkin** [easybuilders/easybuild-easyconfigs#16525]
Hello,

I believe I have found a similar issue to https://github.com/easybuilders/easybuild-easyconfigs/issues/16467#issue-1418126311 in the installation of `CIF2Cell-1.2.10-GCCcore-8.3.0-Python-2.7.16.eb`. In particular, the installation fails with the following pip error:
```
== FAILED: Installation ended unsuccessfully (build directory: /home/nmize/.local/easybuild/build/Python/2.7.16/GCCcore-8.3.0): build failed (first 300 chars): cmd " /opt/modules/software/Python/2.7.16-GCCcore-8.3.0/bin/python -m pip install --prefix=/opt/modules/software/Python/2.7.16-GCCcore-8.3.0  --no-deps  --no-index  --no-build-isolation  ." exited with exit code 1 and output:
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 202 (took 5 mins 24 secs)
== Results of the build can be found in the log file(s) /tmp/eb-ovbm0_vs/easybuild-Python-2.7.16-20221101.164819.gSONt.log
ERROR: Build of /usr/local/easybuild/easyconfigs/p/Python/Python-2.7.16-GCCcore-8.3.0.eb failed (err: 'build failed (first 300 chars): cmd " /opt/modules/software/Python/2.7.16-GCCcore-8.3.0/bin/python -m pip install --prefix=/opt/modules/software/Python/2.7.16-GCCcore-8.3.0  --no-deps  --no-index  --no-build-isolation  ." exited with exit code 1 and output:\nDEPRECATION: Python 2.7 will reach the end of its life on January 1st, 202')
```

The full log file is here 
[easybuild-Python-2.7.16-20221101.164819.gSONt.log](https://github.com/easybuilders/easybuild-easyconfigs/files/9914454/easybuild-Python-2.7.16-20221101.164819.gSONt.log)

This build was performed on `eb` version `4.6.2`

**Comment by boegel** [comment]
The deprecation part is just a warning, the actual problem is this:

```
AttributeError: 'module' object has no attribute '__legacy__'
```

@Ezimkin  This problem is fixed in PR #16490, please try installing Python 2.7.16 using that updated easyconfig by using "`eb --from-pr 16490 --robot`", and let us know if that worked for you...

**Comment by Ezimkin** [comment]
> 

Ah that makes sense. I can confirm now that this issue is fixed when specifying that PR. 

Thank you for the quick response!
